docs/reference/search/request/sort.asciidoc

```diff
@@ -243,9 +243,24 @@ GET /_search
 
     How to compute the distance. Can either be `sloppy_arc` (default), `arc` (slightly more precise but significantly slower) or `plane` (faster, but inaccurate on long distances and close to the poles).
 
-Note: the geo distance sorting supports `sort_mode` options: `min`,
+`sort_mode`::
+
+    What to do in case a field has several geo points. By default, the shortest
+    distance is taken into account when sorting in ascending order and the
+    longest distance when sorting in descending order. Supported values are
+    `min`, `max`, `median` and `avg`.
+
+`unit`::
+
+    The unit to use when computing sort values. The default is `m` (meters).
+
+NOTE: the geo distance sorting supports `sort_mode` options: `min`,
 `max` and `avg`.
```

Comment:
This sentence is a (near) dup of line 251 which also includes "median". Suggest removing it?